src/tiledimage.js

```diff
@@ -533,6 +549,32 @@ $.extend($.TiledImage.prototype, $.EventSource.prototype, /** @lends OpenSeadrag
                 OpenSeadragon.getElementPosition( this.viewer.element ));
     },
 
+    // private
+    // Convert rectangle in tiled image coordinates to viewport coordinates.
+    _tiledImageToViewportRectangle: function(rect) {
+        var scale = this._scaleSpring.current.value;
```

Comment:
> Hmm... and y between 0 and aspect ratio?

Yes!



> Even so, maybe there's a better name

Anything in mind?
Also, it appears I don't need `tiledImageToViewportRectangle` but just `_viewportToTiledImageRectangle`. Do you think I should leave it anyway?